OpenRA.Mods.Cnc/Scripting/Properties/InfiltrateProperties.cs

```diff
@@ -37,7 +37,7 @@ public void Infiltrate(Actor target)
 			if (infiltrates == null)
 				throw new LuaException("{0} tried to infiltrate invalid target {1}!".F(Self, target));
 
-			Self.QueueActivity(new Infiltrate(Self, Target.FromActor(target), infiltrates));
+			Self.QueueActivity(new Infiltrate(Self, Target.FromActor(target), infiltrates, null));
```

Comment:
This and the other Lua API changes will prevent the target lines from being shown. This is a non-trivial behaviour change that wasn't mentioned in the PR description!

Reply:
I understand your logic here based on previous discussions: automatic behaviours should not have target lines - but i'm not sure this is correct here, because player-controlled units may have commands scripted and these *should* display when the user presses the shift key.

Reply:
I'm not sure they should. Scripted behaviours are things that are outside of the player's control (in the general sense) and I think it would be illogical and immersion breaking for the player to have foreknowledge of queued activities that where queued outside of the player's control.

Besides, wouldn't player-control over a scripted unit be inherently contradictory anyway? As far as I remember, the original games would always temporarily remove a unit from the player's control when it needed to perform a scripted action.

Reply:
Ok. Considering that this is currently a hypothetical situation I am okay with letting it be until/if it ever becomes an issue in practice. Silently changing the behaviour is still a problem though, can we please at least mention this in the extended commit message, and ideally split it into its own commit to improve visibility in the git history and blame?